packages/ndla-ui/src/Article/ArticleByline.tsx

```diff
@@ -56,12 +56,13 @@ type AccordionHeaderVariants = 'white' | 'blue';
 type Props = {
   authors?: AuthorProps[];
   suppliers?: SupplierProps[];
-  published: string;
-  license: string;
+  published?: string;
+  license?: string;
   licenseBox?: ReactNode;
   locale?: string;
   footnotes?: FootNote[];
   accordionHeaderVariant?: AccordionHeaderVariants;
+  isFrontpageArticle?: boolean;
```

Comment:
Dette kan på sikt tenkes skal brukes på andre plasser. Tenker det er lurt å være litt generiske her.
```suggestion
  displayByline?: boolean;
```